src/Sentry/Platforms/Android/SentryOptions.cs

```diff
@@ -270,7 +270,7 @@ public void AddInAppExclude(string prefix)
         public bool EnableAndroidSdkTracing { get; set; } = false;
 
         /// <summary>
-        /// Gets or sets a value that indicates if the <see cref="BeforeSend"/> callback will be invoked for
+        /// Gets or sets a value that indicates if the <see cref="o:SetBeforeSend"/> callback will be invoked for
```

Comment:
IMO, It's still the "BeforeSend" callback, but it's set with `SetBeforeSend`.
```suggestion
        /// Gets or sets a value that indicates if the <see cref="o:BeforeSend"/> callback will be invoked for
```